src/mSupplyMobileApp.js

```diff
@@ -38,6 +38,7 @@ import { FirstUsePage } from './pages';
 import { SupplierCredit } from './widgets/modalChildren/SupplierCredit';
 
 import globalStyles, { SUSSOL_ORANGE } from './globalStyles';
+import { TemperatureSyncActions } from './actions/TemperatureSyncActions';
 
 const SYNC_INTERVAL = 10 * 60 * 1000; // 10 minutes in milliseconds.
```

Comment:
this would be good to move to the `MILLISECONDS` const

Reply:
there are quite a few scattered around, I've made an issue: https://github.com/openmsupply/mobile/issues/2663